### PR TITLE
CVE-2005-2970

### PR DESCRIPTION
**Pull request by exw4141**

### CODE REVIEW

File: cves/CVE-2005-2970.yml [hunk 1]
```diff
@@ -43,17 +53,19 @@ bounty:
...
 reviews: []
 bugs: []
-repo:
+repo: https://github.com/apache/httpd
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
+   - commit: 6a28069cef904802aeb334adce5e210a7379194c
+     note:
```
Comment (quinnreed) [inline cves/CVE-2005-2970.yml]: Some commit message or description on what was fixed here would be good

File: cves/CVE-2005-2970.yml [hunk 2]
```diff
@@ -97,22 +110,20 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
+  answer: No evidence found
```
Comment (quinnreed) [inline cves/CVE-2005-2970.yml:113]: Instructions say this can be left blank if no evidence. Feel free to remove

**Comment by quinnreed** [comment]
3 Votes Added

File: cves/CVE-2005-2970.yml (hunk 1 above)
Comment (exw4141) [reply cves/CVE-2005-2970.yml]: Added general description of the fix

File: cves/CVE-2005-2970.yml (hunk 2 above)
Comment (exw4141) [reply cves/CVE-2005-2970.yml:113]: Done

Comment (exw4141) [reply cves/CVE-2005-2970.yml:113]: Content is required to pass Travis so the line was added back into the file

**Pull request merged**